### ISSUE TITLE
Misplaced menu

### ISSUE THREAD

**Issue by psydex** [jshackles/Enhanced_Steam_Firefox#86]
ES 7.3 / Firefox
(8.0.1 doesn't seem to be available for Firefox just yet!?)
![misplaced menu](https://cloud.githubusercontent.com/assets/5594251/10781910/73f575ea-7d54-11e5-9e2f-0d96c8e46292.jpg)

**Comment by Rcannon95** [comment]
Firefox ihas not had the most recent version for several months since he dropped Firefox support. Mozilla is implementing the ability to use Chrome addons for Firefox and he is working on supporting/testing that.

**Comment by psydex** [comment]
But Firefox addons CAN be sideloaded!?!  ಠ_ಠ

**Comment by Rcannon95** [comment]
Yes, but new versions have not been developed and publicly available for Firefox for several months.

**Comment by Rcannon95** [comment]
Sorry, I thought the focus of the question was the top portion of your question, my apologies.
